dist/index.d.ts

```diff
@@ -960,9 +1174,25 @@ interface GPUShaderModuleDescriptorSPIRV
 }
```

Comment:
This deprecated interface can be deleted too, since we're removing the typedef that uses it.

Reply:
Ah yeah, nice catch. Will do 👍